.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Too many changes for one PR. 

Issue/PR templates and workflows could be done in 2 separate PRs